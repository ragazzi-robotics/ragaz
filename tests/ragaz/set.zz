def main():

    # TODO: Make it work well: the problem is that sometimes it prints in a way, sometimes in a completely diferent way
    var a = {1, 2, 3, 4, 101, 5, 6, 1}
    #print(a)
    a.add(9)
    a.remove(1)
    #for element in a:
    #    print(element)

    var b = set([3, 2, 1])
    var it = iter(b)
    print(next(it))

    # TODO: Make it work:
    #var a = {"a", "b", "c"}
    #a.add("d")
    #a.remove("b")
