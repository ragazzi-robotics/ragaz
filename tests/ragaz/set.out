{'1', '2', '3', '4', '6', '101', '5', }
2
3
4
6
9
101
5
1
